src/pages/MindMap/index.tsx

```diff
@@ -41,12 +42,15 @@ const MindMap = () => {
     batch.commit();
   };
   return (
-    <MindMapSimulation
-      data={mindmap}
-      addNode={addNode}
-      deleteNode={deleteNode}
-      updateNode={updateNode}
-    />
+    <div style={{ margin: 0, padding: 0 }}>
+      <MindMapSimulation
+        data={mindmap}
+        addNode={addNode}
+        deleteNode={deleteNode}
+        updateNode={updateNode}
```

Comment:
I have no idea how to call the `addNode` function, so I wrote a generic "call function with parameters" component to let you implement that yourself.